src/System.Security.Cryptography.Csp/src/System.Security.Cryptography.Csp.csproj

```diff
@@ -11,24 +11,26 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net463'">true</IsPartialFacadeAssembly>
     <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' == 'net463'">None</ResourcesSourceOutputDirectory>
+    <NoWarn Condition="'$(TargetsUnix)' == 'true'">CS0809</NoWarn>
```

Comment:
This is to prevent a build error for this auto-generated file. The source and ref files have the corresponding #pragma. Short of maintaining the unsupported.cs file by hand, this seems reasonable especially since it only applies to the Unix\PlatformNotSupported assembly.